ci/management/commands/generate_reports.py

```diff
@@ -405,3 +435,46 @@ def handle_enrollments(self, sheet, sbm_client, ids_client, start_date,
                 5: data[key]['optouts'],
                 6: data[key]['completed'],
             })
+
+    def handle_sms_delivery_msisdn(
+            self, sheet, ms_client, start_date, end_date):
+
+        outbounds = self.get_outbounds(
+            ms_client,
+            created_after=start_date.isoformat(),
+            created_before=end_date.isoformat(),
+            ordering='created_at'
+        )
+
+        data = collections.defaultdict(partial(collections.defaultdict, int))
+        count = collections.defaultdict(int)
+        for outbound in outbounds:
+
+            if 'voice_speech_url' not in outbound.get('metadata', {}):
+
+                count[outbound['to_addr']] += 1
```

Comment:
Do we have any sort of guarantee from the API that this list will be returned in chronological order?